lib/paper_trail/compatibility.rb

```diff
@@ -18,7 +18,7 @@ module PaperTrail
   # versions.
   module Compatibility
     ACTIVERECORD_GTE = ">= 6.1" # enforced in gemspec
-    ACTIVERECORD_LT = "< 7.2" # not enforced in gemspec
+    ACTIVERECORD_LT = "< 8" # not enforced in gemspec
```

Comment:
This should most likely be 7.3, as we're cautious with Rails introducing breaking changes in minor versions.

Reply:
There won't be 7.3, only 8 as a next.

Reply:
I decided to use 8 because looks like there won't be Rails 7.3.

We can see this [here](https://github.com/rails/rails/commit/07d51f9fa3a05386010b640bd121da88c9e341e4) and [here](https://github.com/rails/rails/milestones)

Reply:
BTW, changed it :slightly_smiling_face:

So many people wait for the update :slightly_smiling_face: